includes/ziti/ziti.h

```diff
@@ -420,8 +420,16 @@ extern int ziti_enroll(ziti_enroll_opts *opts, uv_loop_t *loop, ziti_enroll_cb e
 ZITI_FUNC
 extern void ziti_set_app_info(const char *app_id, const char *app_version);
 
+/**
+ * Load ziti identity config from memory or file.
+ * First it tries to parse [conf_str] as identity Json.
+ * if that fails it tries to load it from file using [conf_str] as the path.
```

Comment:
does it allow for relative path from cwd too?

Reply:
yes